### DISCUSSION TITLE
arm64/M1 support for php7.2

### DISCUSSION THREAD

**Discussion by mathias-heilmann** [docksal/docksal#1671] [General]
The latest PHP support for arm64/M1 architecture is the `docksal/cli:php7.3-3.0` image. Is it "easy" to provide an docksal image for php7.2?

Thanks for your answer!

**Comment by lmakarov** [comment]
We won't be officially offering or supporting EOL PHP versions.

PHP 7.2 was dropped over 2 years ago. The last `docksal/cli` release that included it was [v2.11.1](https://github.com/docksal/service-cli/tree/v2.11.1).
Starting with [v3.1.0](https://github.com/docksal/service-cli/releases/tag/v3.1.0), we only support PHP 7.4, PHP 8.0, and PHP 8.1.

Technically, it should be possible to compare the most recent [PHP 7.4 Dockerfile](https://github.com/docksal/service-cli/blob/v3.2.2/7.4/Dockerfile) with the last [PHP 7.2 Dockerfile](https://github.com/docksal/service-cli/blob/v2.11.1/7.2/Dockerfile) and port over the necessary arm64 adjustments (anywhere `TARGETARCH` or `arm64` are mentioned).

If you really need this and are willing to sponsor the development effort, feel free to drop a note at hello@docksal.io.

**Comment by mathias-heilmann** [comment]
Thank you very much, I'll give it a try.
